metals/src/main/scala/scala/meta/internal/builds/GradleBuildTool.scala

```diff
@@ -16,72 +16,28 @@ case class GradleBuildTool() extends BuildTool {
       .map(s => scala.meta.Lit.String(s).syntax)
       .mkString("[", ",", "]")
 
+  private def additionalRepos =
+    if (BuildInfo.gradleBloopVersion.contains("+"))
```

Comment:
can you extract the condition into the variable?

Reply:
Fixed